api/templates/legal-police-additional-offenses.xml

```diff
@@ -16,7 +16,7 @@
         <Charge ID="1">
           <Charge>{{text $Item.CourtCharge}}</Charge>
           <Date Type="{{dateEstimated $Item.Date}}">
```

Comment:
Should this `.Date` be updated to `.CourtDate` as well?

Reply:
Ah, yes – thanks!